src/components/quotes/Quotes.jsx

```diff
@@ -0,0 +1,67 @@
+import React, { useEffect, useState } from 'react';
+import Swal from 'sweetalert2';
+import './quotes.css';
+
+const ShowQuotes = () => {
+  const [quotes, setQuotes] = useState([]);
+
+  useEffect(() => {
+    const apiUrl = process.env.REACT_APP_API_URL;
+    const apiKey = process.env.REACT_APP_API_KEY;
+
+    Swal.fire({
+      title: 'Fetching Quotes...',
+      html: '<div class="sweet-alert-loading-spinner"><div class="spinner"></div></div>',
+      allowOutsideClick: false,
+      showConfirmButton: false,
+    });
```

Comment:
- [ ] Good job so far, :clap: However, Please do not use another third-party library in implementing your fetch, and also it throws an error when I run your code, below is the screenshot. :slightly_smiling_face: :arrow_down: 

![ErrorStudent](https://github.com/lucy-sees/MathMystics/assets/43528594/cb1425b2-654f-4d1a-be27-f1abfbf9a5f5)